mne/cov.py

```diff
@@ -1713,6 +1713,8 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol=1e-4,
         thresholded to determine the rank are also returned.
     """
     picks_list = _picks_by_type(info)
+    if scalings is None:
+        scalings = dict(mag=1e15, grad=1e13, eeg=1e6)
```

Comment:
> @dengemann what do you think? we should use a file global if we use the
same values at 2 locations in the same file

Yes, this would be good. I need to go over the code again and see in which parts which scaling was required. We could have a dict with a reg scaling and a natural scaling. C.f. forthcoming PRNI paper.